components/style/properties/shorthand/position.mako.rs

```diff
@@ -127,3 +127,27 @@
         }
     }
 </%helpers:shorthand>
+
+<%helpers:shorthand name="grid-gap" sub_properties="grid-row-gap grid-column-gap"
+                    spec="https://drafts.csswg.org/css-grid/#propdef-grid-gap">
+  use properties::longhands::{grid_row_gap, grid_column_gap};
+
+  pub fn parse_value(context: &ParserContext, input: &mut Parser) -> Result<Longhands, ()> {
+      let row_gap = grid_row_gap::parse(context, input)?;
+      let column_gap = grid_column_gap::parse(context, input).unwrap_or(row_gap.clone());
```

Comment:
This should use `input.try()`; otherwise the cursor won't reset after a failed parse.